staging/src/k8s.io/code-generator/cmd/client-gen/generators/fake/generator_fake_for_clientset.go

```diff
@@ -139,9 +140,13 @@ func NewClientset(objects ...runtime.Object) *Clientset {
 	cs.discovery = &fakediscovery.FakeDiscovery{Fake: &cs.Fake}
 	cs.AddReactor("*", "*", testing.ObjectReaction(o))
 	cs.AddWatchReactor("*", func(action testing.Action) (handled bool, ret watch.Interface, err error) {
+        var opts metav1.ListOptions
+        if watchActcion, ok := action.(testing.WatchActionImpl); ok {
+            opts = watchActcion.ListOptions
+		}
```

Comment:
supernit: It doesn't impact output, but the indentation here in this github view doesn't look right.  Anything that can be done?

Reply:
+1

Reply:
done, ptal.